lib/impure/re.nim

```diff
@@ -200,27 +200,27 @@ proc findBounds*(s: string, pattern: Regex,
   result = findBounds(cstring(s), pattern, matches, start, s.len)
 
 proc findBounds*(buf: cstring, pattern: Regex,
-                 start = 0, bufSize: int): tuple[first, last: int] =
+                 start = 0, bufSize = 0, flags = int32(0)): tuple[first, last: int] =
```

Comment:
flags is not documented, so adding this optional param would need a doc comment mention.
But exposing a raw int32 param representing a bitfield is not very nim-esque, and instead should be exposed via an enum. But for this PR it doesn't seem needed to add this optional param in the public API; how about add a private findBounds overload with this extra flags param, so that we can decide in future work how to expose is as public API?

Reply:
ok

Reply:
fixed